libs/core/langchain_core/utils/pydantic.py

```diff
@@ -37,13 +58,13 @@ def is_basemodel_subclass(cls: Type) -> bool:
         return False
 
     if PYDANTIC_MAJOR_VERSION == 1:
-        from pydantic import BaseModel as BaseModelV1Proper
+        from pydantic import BaseModel as BaseModelV1Proper  # pydantic: ignore
```

Comment:
woah what's pydantic: ignore

Reply:
It's our own linter to make sure that we don't import pydantic directly